ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -3823,7 +3823,8 @@ let vlan_destroy printer rpc session_id params =
 let tunnel_create printer rpc session_id params =
   let network = Client.Network.get_by_uuid rpc session_id (List.assoc "network-uuid" params) in
   let pif = Client.PIF.get_by_uuid rpc session_id (List.assoc "pif-uuid" params) in
-  let tunnel = Client.Tunnel.create rpc session_id pif network in
+  let protocol = Record_util.tunnel_protocol_of_string (Listext.assoc_default "protocol" params "gre") in
```

Comment:
Could you use `(List.assoc_opt params "protocol" |> Option.value ~default:"gre")`? We're trying to stop using the Listext library.

Reply:
Done! :)